cypress/integration/configuration/shippingMethod.js

```diff
@@ -0,0 +1,184 @@
+        variantsList = variantsListResp;
+      });
+  });
+
+  beforeEach(() => {
+    cy.clearSessionData().loginUserViaRequest();
+    cy.visit(urlList.shippingMethods);
+  });
+
+  it("should display different price for each channel", () => {
+    const shippingName = `${startsWith}${faker.random.number()}`;
+    const defaultChannelPrice = 11;
+    const createdChannelPrice = 7;
+    const createdChannelCurrency = "PLN";
+
+    let shippingMethod;
+    let shippingZone;
+    let createdChannel;
+
+    channelsUtils
+      .createChannel({
+        name: shippingName,
+        currencyCode: createdChannelCurrency
+      })
+      .then(channel => {
+        createdChannel = channel;
+        shippingUtils.createShipping({
+          channelId: defaultChannel.id,
+          name: shippingName,
+          address: plAddress,
+          price: defaultChannelPrice
+        });
+      })
+      .then(
+        ({
+          shippingMethod: shippingMethodResp,
+          shippingZone: shippingZoneResp
+        }) => {
+          shippingZone = shippingZoneResp;
+          shippingMethod = shippingMethodResp;
+          addChannelToShippingMethod(
+            shippingMethod.id,
+            createdChannel.id,
+            createdChannelPrice
+          );
+        }
+      )
+      .then(() => {
+        cy.addAliasToGraphRequest("ShippingZone");
+        cy.contains(shippingZone.name).click();
+        cy.wait("@ShippingZone");
+        selectChannelInHeader(defaultChannel.name);
+        cy.getTextFromElement(SHIPPING_ZONE_DETAILS.shippingRatePriceTableCell)
+          .then(text => {
+            const expectedValue = getFormattedCurrencyAmount(
+              defaultChannelPrice,
+              defaultChannel.currencyCode
+            );
+            expect(text).to.be.eq(expectedValue);
+
+            selectChannelInHeader(createdChannel.name);
+          })
+          .then(() => {
+            cy.getTextFromElement(
+              SHIPPING_ZONE_DETAILS.shippingRatePriceTableCell
+            );
+          })
+          .then(text => {
+            const expectedValue = getFormattedCurrencyAmount(
+              createdChannelPrice,
+              createdChannelCurrency
+            );
+            expect(text).to.be.eq(expectedValue);
+          });
+      });
+  });
+  it("should create price based shipping method", () => {
+    const shippingName = `${startsWith}${faker.random.number()}`;
+
+    createShippingZone(shippingName, warehouse.name, plAddress.countryFullName);
+    createShippingRate(shippingName, price, rateOptions.PRICE_OPTION);
+
+    createCheckout({
+      channelSlug: defaultChannel.slug,
+      email: "test@example.com",
+      variantsList,
+      address: plAddress,
+      auth: "token"
+    }).then(checkout => {
+      const isShippingAvailable = isShippingAvailableInCheckout(
+        checkout,
+        shippingName
+      );
+      /* eslint-disable no-unused-expressions */
```

Comment:
See this:
https://github.com/cypress-io/eslint-plugin-cypress#chai-and-no-unused-expressions

Reply:
ok, I will install plugin and remove eslint comments